src/metric_model_reporter.h

```diff
@@ -88,6 +88,8 @@ class MetricModelReporter {
   // Lookup counter metric by name, and increment it by value if it exists.
   void IncrementCounter(const std::string& name, double value);
   // Increase gauge by value.
+  void SetGauge(const std::string& name, double value);
```

Comment:
Fix comment from "increase gauge by value" to "overwrite gauge to value"

Reply:
Fixed.